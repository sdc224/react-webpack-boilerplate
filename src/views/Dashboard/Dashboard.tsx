import React from "react";

export default function Dashboard() {
	console.log("world");
	return <div>Dashboard</div>;
}
