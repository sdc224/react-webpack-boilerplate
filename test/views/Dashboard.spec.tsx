import React from "react";
import { render, RenderResult } from "@testing-library/react";
import Dashboard from "@views/Dashboard";

let documentBody: RenderResult;

describe("<Dashboard />", () => {
	beforeEach(() => {
		documentBody = render(<Dashboard />);
	});
	it("matches snapshot", () => {
		const { baseElement } = documentBody;
		expect(baseElement).toMatchSnapshot();
	});
});
