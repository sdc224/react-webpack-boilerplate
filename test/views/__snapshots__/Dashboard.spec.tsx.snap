// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard /> matches snapshot 1`] = `
<body>
  <div>
    <p
      class="MuiTypography-root MuiTypography-body1"
    >
      Dashboard
    </p>
  </div>
</body>
`;
